pkg/testutils/sensors/sensors.go

```diff
@@ -29,7 +29,7 @@ func LoadSensor(t *testing.T, sensori sensors.SensorIface) {
 	}
 
 	t.Cleanup(func() {
-		sensor.Unload(true)
+		sensor.Destroy(true)
```

Comment:
Is this a bugfix we need to backport?

Reply:
it matters for following code, I don't think it's a problem for current code